addons/purchase/static/src/xml/purchase_dashboard.xml

```diff
@@ -1,68 +1,76 @@
-                            <a href="#"><t t-esc="values['all_waiting']"/><br/>Waiting</a>
-                        </td>
-                        <td class="o_main o_dashboard_action" title="All Late RFQs" context='{"search_default_late_rfqs": true}'>
-                            <a href="#"><t t-esc="values['all_late']"/><br/>Late</a>
-                        </td>
-                    </tr>
-                </thead>
-                <tbody>
-                    <tr>
-                        <td class="o_text">
-                            <div>My RFQs</div>
-                        </td>
-                        <td class="o_main o_dashboard_action" title="My Draft RFQs" context='{"search_default_draft_rfqs": true, "search_default_my_purchases": true}'>
-                            <a href="#"><t t-esc="values['my_to_send']"/></a>
-                        </td>
-                        <td class="o_main o_dashboard_action" title="My Waiting RFQs" context='{"search_default_waiting_rfqs": true, "search_default_my_purchases": true}'>
-                            <a href="#"><t t-esc="values['my_waiting']"/></a>
-                        </td>
-                        <td class="o_main o_dashboard_action" title="My Late RFQs" context='{"search_default_late_rfqs": true, "search_default_my_purchases": true}'>
-                            <a href="#"><t t-esc="values['my_late']"/></a>
-                        </td>
-                    </tr>
-                </tbody>
-            </table></div>
-
-            <div class="col-sm-7">
-            <table class="table table-sm">
-                <!-- thead needed to avoid list view rendering error for some reason -->
-                <thead>
-                    <tr>
-                        <!-- can't use th tag due to list rendering error when no values in list... -->
-                        <td class="o_text">Avg Order Value (<t t-esc="values['company_currency_symbol']"/>)</td>
-                        <td><span><t t-esc="values['all_avg_order_value']"/></span></td>
-                        <td class="o_text">Purchased Last 7 Days (<t t-esc="values['company_currency_symbol']"/>)</td>
-                        <td><span><t t-esc="values['all_total_last_7_days']"/></span></td>
-                    </tr>
-                </thead>
-                <tbody>
-                    <tr>
-                        <td class="o_text">Lead Time to Purchase</td>
-                        <td><span><t t-esc="values['all_avg_days_to_purchase']"/> &#160;Days</span></td>
-                        <td class="o_text">RFQs Sent Last 7 Days</td>
-                        <td><span><t t-esc="values['all_sent_rfqs']"/></span></td>
-                    </tr>
-                </tbody>
-            </table></div>
-        </div></div>
+        <div class="o_purchase_dashboard py-4 border-bottom bg-white">
```

Comment:
Too much condensed in `md` screen and too much spread apart in `xxl` :/

`md`
---
<img width="600" alt="image" src="https://user-images.githubusercontent.com/8558351/179183070-68f9ae11-ddbb-4f66-ba0c-bd023184d95c.png">

---------


`xxl`
---
<img width="600" alt="image" src="https://user-images.githubusercontent.com/8558351/179185113-0f4cff8e-9a2a-4236-981b-5521fb0befb7.png">

---------

I'd suggest something like this
```xml
           <div class="row justify-content-between gap-3 gap-lg-0 px-4">
                <div class="col-12 col-lg-auto col-xxl-5 flex-grow-1 flex-xxl-grow-0 flex-shrink-0">
                    <div class="grid h-100 gap-1 gap-lg-2">
                        [...]
                    </div>
                </div>
                <div class="col-12 col-lg-auto flex-shrink-0">
                    <div class="grid h-100 gap-1 gap-lg-2">
                        [...]
                    </div>
                </div>
            </div>
```
           
What do you think?

Reply:
Indeed it looks better, I changed it that way :)